src/components/internal/Card/types.ts

```diff
@@ -14,4 +15,7 @@ export interface CardProps {
     buttonAriaLabel?: string;
     classNameModifiers?: string[];
     testId?: string;
+    expandable?: boolean;
 }
+
+export type AriaRole = Pick<JSXInternal.HTMLAttributes<HTMLDivElement>, 'role'>;
```

Comment:
Consider changing this to `JSXInternal.HTMLAttributes<HTMLDivElement>['role']` to get the actual role type, rather than a record with a `role` property.

```suggestion
export type AriaRole = JSXInternal.HTMLAttributes<HTMLDivElement>['role'];
```